### PR TITLE
Add pipeline for updating deprecated classes from external sources

### PR DESCRIPTION
**Pull request by matentzn**
- [ ] Document `update_deprecated_mappings` command in Mondo editor docs

### CODE REVIEW

File: src/ontology/mondo.Makefile [hunk 1]
```diff
@@ -640,6 +641,21 @@ merge_obsolete_template: tmp/heal_hierarchy.ru $(MERGE_TEMPLATE) tmp/remove_clas
...
+update_deprecated_mappings: $(ALL_SOURCES_DEPRECATED_PATTERNS)
+	$(ROBOT) template --prefix "CHR: http://purl.obolibrary.org/obo/CHR_" --prefix "sssom: https://w3id.org/sssom/" --merge-before --input $(SRC) \
+ 		$(patsubst %, --template %, $^) \
+ 		convert -f obo -o $(SRC)
+	$(MAKE) NORM && mv NORM $(SRC)
+
+deprecated_annotation_merging:
+	sed -i 's/source="MONDO:equivalentObsolete",\ source="MONDO:obsoleteEquivalentObsolete"/source="MONDO:obsoleteEquivalentObsolete"/g' mondo-edit.obo || true
+	sed -i 's/source="MONDO:equivalentObsolete",\ source="MONDO:equivalentTo"/source="MONDO:equivalentObsolete"/g' mondo-edit.obo || true
+	#(.*)source="MONDO:equivalentObsolete"(.*)source="MONDO:obsoleteEquivalentObsolete"(.*) //
```
Comment (matentzn) [inline src/ontology/mondo.Makefile:658]: @joeflack4, can you turn this comment into an `sed` command? I provide some pseudocode below. This does not work, and also causes a bug if the second `source=` block is the last in the OBO xref (e.g. `xref: OMIM:601251 {source="MONDO:equivalentObsolete", source="GARD:0010117", source="MONDO:obsoleteEquivalentObsolete"}`). We do _not_ want the `,` before the `}` symbol. 

```suggestion
	sed -i 's/(.*)source="MONDO:equivalentObsolete"(.*)source="MONDO:obsoleteEquivalentObsolete"(.*)/$1source="MONDO:obsoleteEquivalentObsolete"$2$3/g' mondo-edit.obo || true
```

Comment (matentzn) [reply src/ontology/mondo.Makefile:658]: Closed in favour of #6345

**Comment by matentzn** [comment]
ready for merging after QC passed.

**Review by nicolevasilevsky — APPROVED** [review]

**Comment by matentzn** [comment]
@nicolevasilevsky did you check the action item [above](https://github.com/monarch-initiative/mondo/pull/6344#issue-1751453682)?

**Comment by nicolevasilevsky** [comment]
yes, I added it to our 1:1 agenda

**Pull request merged**